pkg/syncer/statussyncer.go

```diff
@@ -64,6 +70,13 @@ func NewStatusSyncer(from, to *rest.Config, gvrs []string, kcpClusterName logica
 	return New(kcpClusterName, pclusterID, fromClient, toClient, SyncUp, gvrs, pclusterID, mutatorsMap)
 }
 
+func (c *Controller) deleteFromUpstream(ctx context.Context, gvr schema.GroupVersionResource, upstreamNamespace, name string) error {
```

Comment:
move `if advancedSchedulingFeatureEnabled {` up to where `c.deleteFn` is called, with a closure.

Reply:
done